Bugsnag/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry_MachException.c

```diff
@@ -45,12 +45,19 @@
 #define kThreadPrimary "KSCrash Exception Handler (Primary)"
 #define kThreadSecondary "KSCrash Exception Handler (Secondary)"
 
+#if __LP64__
```

Comment:
We probably don't even need this kind of thing anymore. Are there any 32-bit targets that are supported by the latest Xcode?

Reply:
The latest Xcode does not support 32-bit targets, but our [README](https://github.com/bugsnag/bugsnag-cocoa/blob/master/README.md) states we support iOS 9.3+, macOS 10.11+ and tvOS 9.2+, which still include 32-bit support 🤔

Reply:
But if Xcode doesn't support 32-bit, then nobody can compile 32-bit applications anymore for submission to the Apple stores (they only accept submissions compiled on the latest Xcode). Maybe we should update our supported targets list?

Reply:
Agreed, all iOS developers must be using the latest Xcode to get on the App Store. It's possible there could be a few developers using old versions of Xcode to build with old macOS SDKs.

Perhaps it's worth us documenting which Xcode versions we support building with?

Reply:
Resolving this as it's been raised as a separate issue ROAD-1117

Reply:
New info: Nick discovered that the watches still use armv7 chips (32-bit). So we can't drop 32-bit support yet, especially given the newer watches with networking support.